lib/modules/auth/login/login_screen.dart

```diff
@@ -1,232 +1,213 @@
-                        ConditionalBuilder(
-                          condition: state is! LoginLoadingState,
-                          builder: (context) {
-                            return Container(
-                              height: 45,
-                              width: double.infinity,
-                              child: ElevatedButton(
-                                onPressed: () {
-                                  if (formKey.currentState.validate()) {
-                                    LoginModel loginModel = LoginModel(
-                                      email: emailController.text.trim(),
-                                      password: passwordController.text.trim(),
-                                    );
-                                    LoginCubit.get(context).userLogin(
-                                      loginModel: loginModel,
-                                    );
-                                  }
-                                },
-                                child: Text("LOGIN"),
-                              ),
-                            );
-                          },
-                          fallback: (context) => buildLoadingState(),
-                        ),
-                        SizedBox(height: 16),
-                        Row(
-                          mainAxisAlignment: MainAxisAlignment.center,
-                          children: [
-                            Text("Don't have an account?"),
-                            TextButton(
-                                onPressed: () =>
-                                    navigateToRegisterScreen(context),
-                                child: Text(
-                                  "Register",
-                                  style: TextStyle(
-                                      fontWeight: FontWeight.bold,
-                                      fontSize: 14),
-                                )),
-                          ],
-                        )
-                      ],
-                    ),
-                  ),
-                ),
-              ),
-            ),
-          );
-        },
+    return BlocProvider<LoginBloc>(
+      create: (context) => LoginBloc(LoginRepository()),
+      child: Scaffold(
+        appBar: AppBar(),
+        body: BlocConsumer<LoginBloc, LoginState>(
+          listener: (context, state) {
+            if (state is LoginSuccessState) {
+              buildToastMessage(state.model.message, Colors.green);
+              navigateToHomeScreen(context);
+            } else if (state is LoginErrorState) {
+              buildToastMessage(state.message, Colors.red);
+            } else if (state is LoginNavigationToRegisterScreenState) {
+              navigateToRegisterScreen(context);
+            }
+          },
+          builder: (context, state) {
+            if (state is LoginInitialState) {
+              return initialWidget(context);
```

Comment:
change name of this widget (initialWidget)

Reply:
to loginInitialWidget ?